specs/release-automation/post-merge-automation-workflow.puml

```diff
@@ -0,0 +1,28 @@
+@startuml
+title Post merge workflow
+:Dev branch release PR merged to main branch;
+start
+group RA-4: Deploying the files to OSF
+    if (Is there previous release on OSF) then (no)
+        :Push files in the OSF staging directory to OSF using the user's OSF key;
```

Comment:
Did you investigate if you can store files in an action?

Reply:
Yes and they can be stored and access across different jobs. And even non github action jobs can access them.